TADProc-Tests.postman_collection.json

```diff
@@ -0,0 +1,135 @@
            \"host\": \"http://openfaas:8080\",\n                                        \"cfg\": \"1.0\"\n                                    }\n                                ]\n                            }\n                        ]\n                    },\n                    {\n                        \"id\": \"002@1.0\",\n                        \"host\": \"http://openfaas:8080\",\n                        \"cfg\": \"1.0\",\n                        \"typologies\": [\n                            {\n                                \"id\": \"030@1.0\",\n                                \"host\": \"https://frmfaas.sybrin.com/function/off-typology-processor\",\n                                \"cfg\": \"030@1.0\",\n                                \"rules\": [\n                                    {\n                                        \"id\": \"003@1.0\",\n                                        \"host\": \"https://frmfaas.sybrin.com/function/off-rule-003\",\n                                        \"cfg\": \"1.0\"\n                                    },\n                                    {\n                                        \"id\": \"006@1.0\",\n                                        \"host\": \"http://openfaas:8080\",\n                                        \"cfg\": \"1.0\"\n                                    }\n                                ]\n                            },\n                            {\n                                \"id\": \"031@1.0\",\n                                \"host\": \"https://frmfaas.sybrin.com/function/off-typology-processor\",\n                                \"cfg\": \"031@1.0\",\n                                \"rules\": [\n                                    {\n                                        \"id\": \"003@1.0\",\n                                        \"host\": \"https://frmfaas.sybrin.com/function/off-rule-003\",\n                                        \"cfg\": \"1.0\"\n                                    },\n                                    {\n                                        \"id\": \"007@1.0\",\n                                        \"host\": \"http://openfaas:8080\",\n                                        \"cfg\": \"1.0\"\n                                    }\n                                ]\n                            }\n                        ]\n                    }\n                ]\n            }\n        ]\n    },\n    \"ruleResults\": [\n        {\n            \"rule\": \"003@1.0\",\n            \"result\": true,\n            \"reason\": \"asdf\",\n            \"subRuleRef\": \"123\"\n        },\n        {\n            \"rule\": \"028@1.0\",\n            \"result\": true,\n            \"subRuleRef\": \"04\",\n            \"reason\": \"The debtor is 50 or older\"\n        }\n    ],\n    \"typologyResult\": {\n        \"typology\": \"028@1.0\",\n        \"result\": 50\n    },\n    \"channelResult\": { \"result\": 0, \"channel\": \"001@1.0\" }\n}",
```

Comment:
please replace https://frmfaas.sybrin.com with http://gateway.openfaas:8080 (even in the network sub-map) for ALL future tests..........................